parquet/src/test/java/org/apache/iceberg/parquet/TestParquet.java

```diff
@@ -51,8 +54,15 @@
 
   @Test
   public void testRowGroupSizeConfigurable() throws IOException {
-    // Without an explicit writer function
-    File parquetFile = generateFileWithTwoRowGroups(null).first();
+    Map<String, String> props = ImmutableMap.of(
+        PARQUET_ROW_GROUP_SIZE_BYTES,
+        Integer.toString(4 * Integer.BYTES));
+    // Without an explicit writer function doesn't support PARQUET_ROW_GROUP_CHECK_MIN_RECORD_COUNT
+    // PARQUET_ROW_GROUP_CHECK_MAX_RECORD_COUNT configs.
+    // Even though row group size is 16 bytes, we still have to write 101 records
+    // as default PARQUET_ROW_GROUP_CHECK_MIN_RECORD_COUNT is 100.
+    File parquetFile = generateFileWithTwoRowGroups(null, 101, props)
+        .first();
```

Comment:
Does this need to be on a separate line?

Reply:
fixed